pkg/kubelet/kubelet.go

```diff
@@ -1349,6 +1405,33 @@ func (kl *Kubelet) initializeModules() error {
 		kl.serverCertificateManager.Start()
 	}
 
+	if kl.dynamicCertificateController != nil {
+		// runonce to try to prime data.  If this fails, it's ok because we fail closed.
+		// Files are required to be populated already, so this is for convenience.
+		if err := kl.dynamicCertificateController.RunOnce(); err != nil {
+			klog.Warningf("Initial population of dynamic certificates failed: %v", err)
+		}
+		go kl.dynamicCertificateController.Run(1, stopCh)
```

Comment:
I think we want to start the clientCertificateCAContentProvider and certKeyContentProvider first otherwise the runonce will load nils from the two providers

Reply:
Done.